assets/js/components/ClusterDetails/ChecksResultOverview.jsx

```diff
@@ -55,10 +55,18 @@ const ChecksResultOverview = ({
       </h6>
 
       <div className="flex flex-col self-start w-full px-4 mt-2">
-        <CheckResult onClick={onCheckClick} value={passing} result="passing" />
-        <CheckResult onClick={onCheckClick} value={warning} result="warning" />
         <CheckResult
-          onClick={onCheckClick}
+          onClick={() => onCheckClick('passing')}
```

Comment:
I see single and double quotes mixed here hehe
Can we use the same? XD

Reply:
I actually used double quotes but the linter on autosave changed them for single quotes

Reply:
Yeah single quotes are fine here, don't worry, Prettier is always right :smile: